src/utils/token.js

```diff
@@ -0,0 +1,11 @@
+export const setToken = (res) => {
+  const accessToken = res.data.accessToken;
+  const refreshToken = res.data.refreshToken;
+  localStorage.setItem("accessToken", accessToken);
+  localStorage.setItem("refreshToken", refreshToken);
+};
```

Comment:
res 전체를 받아서 접근하는 지금의 방식이라면, 추후에 res 형태가 바뀌었을 때 대응하지 못할 것 같습니다. 예를 들어 res.accessToken과 같이 depth가 줄어든다고 했을 때 여기서 에러가 발생하겠네요.

accessToken과 refreshToken 인자 두 개를 받는 방식이 더 좋을 것 같습니다.

```suggestion
export const setToken = (accessToken, refreshToken) => {
  localStorage.setItem("accessToken", accessToken);
  localStorage.setItem("refreshToken", refreshToken);
};
```